src/common/libjsc/jstatctl.c

```diff
@@ -1144,19 +1147,19 @@ static int notify_status_obj (flux_t *h, jsc_handler_obj_f func, void *d)
         rc = -1;
         goto done;
     }
-    if (flux_msg_handler_addvec (h, htab, (void *)ctx) < 0) {
+    if (flux_msg_handler_addvec (h, htab, NULL, &handlers) < 0) {
```

Comment:
The elimination of (void *)ctx doesn't appear to be related to this logical commit.

Reply:
Yes, will split out.